connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -2306,6 +2306,9 @@ void Gap::signal_connection_complete(
         }
     }
 #else
+    if (!apply_peripheral_privacy_connection_policy(event)) {
```

Comment:
That will cause an issue if `BLE_FEATURE_PRIVACY ` is not set as `apply_peripheral_privacy_connection_policy ` won't be available.

Reply:
thanks